### ISSUE TITLE
Migrate to AppNavigation components from vue-collection

### ISSUE THREAD

**Issue by georgehrke** [nextcloud/calendar#1495]
Currently missing in vue collection:
- No way to provide Share icon and shared / published indicator.
- No way to provide avatar of calendar owner
- No way to add  Share dialog
- There is no ColorPicker Action yet.

**Comment by skjnldsv** [comment]
> * No way to provide avatar of calendar owner

``` vue
<AppNavigationItem title="This is bullet item">
	<Avatar slot="icon" user="admin" />

```
> * There is no ColorPicker Action yet.

Use the actionInput type="color" for now, it will be automatically updated later

**Comment by georgehrke** [comment]
```
<AppNavigationItem title="This is bullet item">
	<Avatar slot="icon" user="admin" />
```

This would replace the colored icon in front, so you don't know anymore what color the calendar has.

**Comment by skjnldsv** [comment]
> This would replace the colored icon in front, so you don't know anymore what color the calendar has.

Ah yes, sorry! I misunderstood the sentence! :)

**Comment by georgehrke** [comment]
This was already implemented with #926 :)
